### PR TITLE
docs: example non-linear histogram

### PR DESCRIPTION
**Pull request by kanitw**
Thanks @Saba9

### CODE REVIEW

File: examples/compiled/histogram_nonlinear.vg.json [hunk 1]
```diff
@@ -0,0 +1,120 @@
+{
+  "$schema": "https://vega.github.io/schema/vega/v5.json",
+  "description": "A histogram for a data source that provides non-linear bins. Thanks to [@Saba9](https://github.com/Saba9) who helps create this example.",
```
Comment (domoritz) [inline examples/compiled/histogram_nonlinear.vg.json:3]:
```suggestion
  "description": "A histogram for a data source that provides non-linear bins. Thanks to [@Saba9](https://github.com/Saba9) who helped create this example.",
```

File: examples/specs/histogram_nonlinear.vl.json [hunk 2]
```diff
@@ -0,0 +1,72 @@
...
+      },
+      {
+        "startTime": "8.33",
+        "endTime": "12.50",
+        "residency": 0
+      },
+      {
+        "startTime": "12.50",
+        "endTime": "16.67",
+        "residency": 31.17
```
Comment (domoritz) [inline examples/specs/histogram_nonlinear.vl.json:19]: If you convert this to actual ratios, we can use the format `.2f%` below.

File: examples/specs/histogram_nonlinear.vl.json [hunk 3]
```diff
@@ -0,0 +1,72 @@
...
+      "sort": null
+    },
+    "x2": {"field": "endTime"},
+    "y": {
+      "title": "",
+      "field": "residency",
+      "type": "quantitative",
+      "scale": {"domain": [0, 100]},
+      "axis": {
+        "labelExpr": "datum.label + '%'"
```
Comment (domoritz) [inline examples/specs/histogram_nonlinear.vl.json:68]: Switch to using format `.2f%`

**Review by domoritz — CHANGES_REQUESTED** [review]

File: examples/specs/histogram_nonlinear.vl.json (hunk 2 above)
Comment (kanitw) [reply examples/specs/histogram_nonlinear.vl.json:19]: what do you mean?

File: examples/specs/histogram_nonlinear.vl.json (hunk 3 above)
Comment (kanitw) [reply examples/specs/histogram_nonlinear.vl.json:68]: There are cases where input data is already in percentage, so I don't think I agree we need to do this here.  I'll merge for now.  If you feel strongly about this, feel free to submit a PR.

**Pull request merged**